OpenRA.Game/Traits/Player/PlayerExperience.cs

```diff
@@ -0,0 +1,35 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Linq;
+
+namespace OpenRA.Traits
+{
+	public class PlayerExperienceInfo : ITraitInfo
```

Comment:
Can you add a `[Desc("")]` here for https://github.com/OpenRA/OpenRA/wiki/Traits documentation?